### PR TITLE
release-0.25 backport security fixes 

### PR DESCRIPTION
**Pull request by pgvishnuram**
## Description

image | old | new
--    | --  | --
ap-base | 3.16.2  | 3.16.2-1
ap-registry |  3.16.2 | 3.16.2-1
ap-db-bootstrapper | 0.26.9 |  0.26.11
ap-elasticsearch | 7.17.5 | 7.17.6
ap-kibana  | 7.17.5 | 7.17.6   
ap-curator | 5.8.4-18 | 5.8.4-19
ap-fluentd | 1.15.1 | 1.15.2
ap-nats-server | 2.8.1-2 | 2.8.1-3
ap-nats-exporter | 0.10.0 | 0.10.0-1
ap-blackbox-exporter | 0.21.1-2 | 0.22.0 
ap-node-exporter  | 1.2.2 | 1.4.0
ap-prometheus | 2.37.0 | 2.37.1
ap-nats-streaming | 0.24.5-1 | 0.24.5-3
ap-auth-sidecar | | 1.23.1-1
ap-configmap-reloader | 0.7.1 | 0.8.0

Additional Updates
replaced nginxinc/nginx-unprivileged:latest  -> quay.io/astronomer/ap-auth-sidecar:1.23.1-1
replaced alpine:3.14 -> alpine:3.16
replaced quay.io/prometheus/node-exporter:v1.2.2 -> quay.io/astronomer/ap-node-exporter:1.4.0

## Related Issues

https://github.com/astronomer/issues/issues/5065

## Testing

Do not merge this PR until this text is replaced with details about how these changes were tested.

## Merging

Do not merge this PR until it lists which release branches this PR should be merged / cherry-picked into.

### CODE REVIEW

**Comment by pgvishnuram** [comment]
@vishwas-astro commander,astro-ui and houston images are pending - will release new images for that as well

File: values.yaml [hunk 1]
```diff
@@ -88,8 +88,8 @@ global:
   # Deploy auth sidecar to use openshift native features
   authSidecar:
     enabled: false
-    repository: nginxinc/nginx-unprivileged
-    tag: stable
+    repository: quay.io/astronomer/ap-auth-sidecar
```
Comment (vishwas-astro) [inline values.yaml:91]: Are we ready to make this change in 0.25 release? What is the risk involved?

Comment (pgvishnuram) [reply values.yaml:91]: Yes since the ap-auth-sidecar  is built on top of nginxinc/nginx-unprivileged and we are safe to make this change

**Review by danielhoherd — APPROVED** [review]
LGTM. Happy to see us switching to ap-node-exporter.

**Comment by vishwas-astro** [comment]
@pgvishnuram do we need a bump in ap-astro-ui and ap-houston-api?

**Pull request merged**